xicam/core/intents/__init__.py

```diff
@@ -55,3 +55,18 @@ def canvas_name(self):
             y_name = self.labels.get("left", "")
             return x_name + ", " + y_name
         return self._canvas_name
+
+
+class ErrorBarIntent(Intent):
+    """
+    For reference on kwargs, see
+    https://pyqtgraph.readthedocs.io/en/latest/graphicsItems/errorbaritem.html
+    """
+
+    canvas = "plot_canvas"
+
+    def __init__(self, labels, **kwargs):
+        super(ErrorBarIntent, self).__init__(**kwargs)
+        self.labels = labels
+        self.x = kwargs['x']
+        self.y = kwargs['y']
```

Comment:
do we need to provide some default match_key like `PlotIntent` does by hashing the labels?

Reply:
I switched to inheriting from plotintent; now that's free

Reply:
:+1: